ChattoAdditions/Source/Chat Items/BaseMessage/Views/BaseMessageCollectionViewCell.swift

```diff
@@ -258,14 +263,40 @@ open class BaseMessageCollectionViewCell<BubbleViewType>: UICollectionViewCell,
     }
 
     // MARK: layout
+
+    private var didAddConstraintsForBubbleView = false
+    private func addBubbleViewConstraintsIfNeeded() {
+        guard !self.didAddConstraintsForBubbleView, let viewModel = self.messageViewModel, let style = self.baseStyle else { return }
+        guard self.useAutolayoutForBubbleView else { return }
+        self.didAddConstraintsForBubbleView = true
+        let layoutConstants = style.layoutConstants(viewModel: viewModel)
+        let percentage = layoutConstants.maxContainerWidthPercentageForBubbleView
+        let offset = layoutConstants.horizontalMargin + layoutConstants.horizontalInterspacing
+        let xConstraint: NSLayoutConstraint
+        if viewModel.isIncoming {
+            xConstraint = bubbleView.leftAnchor.constraint(equalTo: contentView.leftAnchor, constant: offset)
```

Comment:
Is it intentionally that you use left/right anchors instead of leading/trailing? Could be useful for users who need Chatto with RTL languages.

Reply:
We currently don't support that for manual layout, so yeah, I've hardcoded left and right here as well, so it matches manual one